Gemfile

```diff
@@ -77,7 +77,7 @@ group :development, :devunicorn, :test do
   gem 'binding_of_caller'
   gem 'byebug'
   gem 'guard-cucumber'
-  gem 'guard-jasmine', '~> 3.1'
+  # gem 'guard-jasmine', '~> 3.1'
```

Comment:
Is this gem no longer required?

Reply:
The is the gem that prevents upgrading to Rails 6.1. See https://github.com/guard/guard-jasmine/issues/206

If it is required then it must be downgraded to version 1.19.2, and that is only because it is the last version that did not restrict the version of `thor`.

Reply:
Does anyone actually use Guard on your team?  I know a dev prior to me used it and I got in the habit to reduce overhead on running the test suite locally... having said that, I'm not sure I ever ran the jasmine tests via guard 🤷‍♂️

Reply:
yeah, we have agreed to remove for the time being, especially as looking to move to cypress to some extent anyway

Reply:
I (we) can still run jasmine test using bundle instead of guard, so i'm happy to remove it, especially consider no one uses it.